pymongo/srv_resolver.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, fqdn, connect_timeout=None):
 
     def get_options(self):
         try:
-            results = resolver.query(self.__fqdn, 'TXT',
+            results = resolver.resolve(self.__fqdn, 'TXT',
```

Comment:
dns.resolver.resolve was only added in dnspython version 2.0 but pymongo is compatible with "dnspython>=1.16.0,<2.0.0". To make this work with dnspython 1.X we should do this:
```python
try:
    try:
        # dnspython >= 2
        from dns.resolver import resolve as _resolve
    except ImportError:
        # dnspython 1.X
        from dns.resolver import query as _resolve
    _HAVE_DNSPYTHON = True
except ImportError:
    _HAVE_DNSPYTHON = False
```

Reply:
Oops, should have checked that. 😓